build_region.R

```diff
@@ -50,42 +53,46 @@ zones <- ukmsoas[ukmsoas@data$geo_code %in% cents$geo_code, ]
 # load flow dataset, depending on availability
 if(!exists("flow_nat"))
   flow_nat <- readRDS(file.path(pct_bigdata, "pct_lines_oneway_shapes.Rds"))
-summary(flow_nat$dutch_slc / flow_nat$All)
+summary(flow_nat$dutch_slc / flow_nat$all)
 
-if(!exists("rf_nat"))
+if(!exists("rf_nat")){
   rf_nat <- readRDS(file.path(pct_bigdata, "rf.Rds"))
-if(!exists("rq_nat"))
+  rf_nat <- remove_cols(rf_nat, "(waypoint|co2_saving|calories|busyness|plan|start|finish)")
```

Comment:
Ideally this would be done upstream BUT the data is more useful with co2_saving|calories|busyness in.  The others look pretty useless to me.  Thoughts?